src/quo/components/wallet/wallet_overview/style.cljs

```diff
@@ -2,12 +2,13 @@
   (:require
     [quo.foundations.colors :as colors]))
 
-(def container-info
+(defn container-info
+  [{:keys [metrics]}]
   {:padding-horizontal 20
    :padding-top        12
    :padding-bottom     32
    :flex-grow          1
-   :max-height         98})
+   :height             (if (= metrics :none) 78 98)})
```

Comment:
Where does these numbers come from?

It'd be great to also test it on iOS 👍

Reply:
@ulisesmac It comes from Figma.

I tested on iOS and it renders correctly 👍

Reply:
Added iOS demo to the description